go/common/log/log.go

```diff
@@ -16,9 +16,10 @@ func init() { //nolint:gochecknoinits
 
 // Errors will always show
 const (
-	InfoLevel  = zerolog.InfoLevel
-	DebugLevel = zerolog.DebugLevel
-	TraceLevel = zerolog.TraceLevel
+	InfoLevel     = zerolog.InfoLevel
+	DebugLevel    = zerolog.DebugLevel
+	TraceLevel    = zerolog.TraceLevel
+	DisabledLevel = zerolog.Disabled
```

Comment:
@otherview Can you add a description of this log level? e.g. does it suppress error/fatal? It's not a log level I'm familiar with.

Reply:
Can do 👍